src/Toolkit/I18n.php

```diff
@@ -225,6 +247,13 @@ public static function translateCount(string $key, int $count, string $locale =
                 $message = $translation[2];
         }
 
+        if (!$locale) {
+            $locale = static::locale();
+        }
```

Comment:
This should be moved to the beginning of the method for consistency and written like this:

```php
$locale = $locale ?? static::locale();
```